uniprot-search/src/main/resources/proteome/search-fields.json

```diff
@@ -0,0 +1,49 @@
+[
+  {
+    "field": "upid",
+    "fieldValidRegex": "UP[0-9]{9}"
+  },
+  {
+    "field": "proteome_type"
+  },
+  {
+    "field": "annotation_score"
```

Comment:
annotation_score is a sort field as well.

Reply:
done -- gosh, i missed all the sorts for proteome :scream: